models/scholarship.py

```diff
@@ -0,0 +1,39 @@
+import mongoengine as me
+
+class Scholarship(me.Document):
+    id = me.StringField(primary_key=True)
+
+    # eg. Geoffrey F. Butler Award
+    title = me.StringField()
+
+    description = me.StringField()
+
+    # e.g. ["Canadian citizen", "Permanent resident"]
+    citizenship = me.ListField(me.StringField())
+
+    # e.g. ["Arts", "Drama and Speech Communication", "Fine Arts", "Music"]
+    programs = me.ListField(me.StringField())
+
+    eligibility = me.ListField(me.StringField())
+
+    instructions = me.ListField(me.StringField())
+
+    enrollment_year = me.ListField(me.StringField())
+
+    contact = me.StringField()
+
+    link = me.StringField()
+
+    def to_dict(self):
+        """Returns information about the scholarship that is sent
+        to be rendered by Backbone
```

Comment:
You can just say something like "Serialize data to be sent to the client." so as to avoid specifying the frontend technology used (Backbone, React, etc.)